services/ui/ws/window_server.cc

```diff
@@ -507,18 +508,44 @@ void WindowServer::SetPaintCallback(
   window_paint_callback_ = callback;
 }
 
-void WindowServer::StartMoveLoop(uint32_t change_id,
+bool WindowServer::StartMoveLoop(uint32_t change_id,
                                  ServerWindow* window,
                                  WindowTree* initiator,
-                                 const gfx::Rect& revert_bounds) {
+                                 const gfx::Rect& revert_bounds,
+                                 const gfx::Vector2d& drag_offset) {
   current_move_loop_.reset(
       new CurrentMoveLoopState{change_id, window, initiator, revert_bounds});
+
```

Comment:
Ideally, WindowServer should not know about ws::PlatformDIsplay, and route things through the ws::Display.

Reply:
ok, let me then modify other callers and this one in the follow ups!